<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointInPolygon</class>
 <widget class="QWidget" name="PointInPolygon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point In A Polygon</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="xPoint">
     <property name="minimum">
      <double>-9999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QTableWidget" name="PolygonTable">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <column>
      <property name="text">
       <string>Mina</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ponto</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>X</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Y</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QDoubleSpinBox" name="yPoint">
     <property name="minimum">
      <double>-9999999.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="verifyPointButton">
     <property name="text">
      <string>Verificar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Novo Ponto</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Mina:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mine"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Ponto:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="point"/>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QDoubleSpinBox" name="yNewPoint">
        <property name="minimum">
         <double>-9999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QDoubleSpinBox" name="xNewPoint">
        <property name="minimum">
         <double>-9999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="7" rowspan="2">
       <widget class="QPushButton" name="addPointButton">
        <property name="text">
         <string>Adicionar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
